geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/data/RemoveSchemaCommand.scala

```diff
@@ -46,7 +46,7 @@ trait RemoveSchemaCommand[DS <: GeoMesaDataStore[_, _, _ ,_]] extends DataStoreC
         } else {
           println(s"Removing '$typeName'")
           ds.removeSchema(typeName)
-          if (ds.getSchema(typeName) != null) {
+          if (ds.getTypeNames.contains(typeName)) {  // NB: For Cassandra this only checks metadata
```

Comment:
I think in our non-ContentDataStores, we mimicked the behavior of JdbcDataStore, which returns null, even though the javadocs for DataStore seem to indicate it should throw an exception... let's make it:
`if (try { ds.getSchema(typeName) != null } catch { case _: IOException => false })`
that should cover both cases...